app/components/log-content.js

```diff
@@ -14,35 +13,35 @@ import { inject as service } from '@ember/service';
 import { computed } from '@ember/object';
 import { alias, and } from '@ember/object/computed';
 
+const SELECTORS = {
+  CONTENT: '.log-body-content',
+  FIRST_HIGHLIGHT: '.log-line:visible.highlight:first'
+};
+
 Log.LIMIT = config.logLimit;
 
 Log.Scroll = function (options = {}) {
   this.beforeScroll = options.beforeScroll;
   return this;
 };
 
-Log.Scroll.prototype = $.extend(new Log.Listener(), {
+Log.Scroll.prototype = Log.extend(new Log.Listener(), {
```

Comment:
Since other parts of `log-content` use `Log.extend` in this way, I followed that convention here.